docs/getting-started-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,511 @@
+|                                      |     +------------------------------------+    |   +---> |  MySQL ("customer" schema)  | <---+      |
+|                                      |                                               |   |     +-----------------------------+     |      |
+|  +-------------+      +---------+    |     +------------------------------------+    |   |                                         |      |
+|  | OLAP client | ---> | Service | ---+---> | ScalarDB Analytics with PostgreSQL | ---+---+                                         +---+  |
+|  +-------------+      +---------+    |     +------------------------------------+    |   |                                         |   |  |
+|                                      |                                               |   |     +-----------------------------+     |   |  |
+|                                      |     +------------------------------------+    |   +---> | PostgreSQL ("order" schema) | <---+   |  |
+|                                      +---> | ScalarDB Analytics with PostgreSQL | ---+         +-----------------------------+         |  |
+|                                            +------------------------------------+                                                      |  |
+|                                                                                                                                        |  |
+|  +-------------+                                                                                                                       |  |
+|  | OLTP client | ---(Load sample data with a test OLTP workload)-----------------------------------------------------------------------+  |
+|  +-------------+                                                                                                                          |
+|                                                                                                                                           |
++-------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+## Step 1. Start a Kubernetes cluster
+
+First, you need to prepare a Kubernetes cluster. If you use a **minikube** environment, please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md). If you have already started a Kubernetes cluster, you can skip this step.
+
+## Step 2. Start MySQL and PostgreSQL pods
+
+ScalarDB including ScalarDB Analytics with PostgreSQL can use several types of database systems as a backend database. In this document, we use MySQL and PostgreSQL.
+
+You can deploy MySQL and PostgreSQL on the Kubernetes cluster as follows.
+
+1. Add the Bitnami helm repository and update it.
+
+   ```console
+   helm repo add bitnami https://charts.bitnami.com/bitnami
+   ```
+
+   ```console
+   helm repo update bitnami
+   ```
```

Comment:
> Or maybe the following (combining the commands) could work?

Yes. It works.
However, I will apply the first one because I want to make these steps one-by-one as much as possible.